src/coreComponents/mesh/docs/Mesh.rst

```diff
@@ -206,14 +206,14 @@ The ``ElementRegions`` are defined as below :
 .. code-block:: xml
 
   <ElementRegions>
-    <ElementRegion
+    <CellElementRegion
       name="Top"
-      cellBlocks="Top_hexahedra Top_wedges Top_tetrahedra"
-      materialList="water rock"/>
-    <ElementRegion
+      cellBlocks="{ Top_hexahedra, Top_wedges, Top_tetrahedra }"
+      materialList="{ water, rock }"/>
+    <CellElementRegion
       name="Bot"
-      cellBlocks="Bot_hexahedra Bot_wedges Bot_tetrahedra"
-      materialList="water rock"/>
+      cellBlocks="{ Bot_hexahedra, Bot_wedges, Bot_tetrahedra }"
```

Comment:
```suggestion
      cellBlocks="{ hexahedra, wedges, tetrahedra, pyramids }"
```

When we removed the old mesh importer, the naming convention of the importing cell block changed. It is no longer possible to define a name (like `Bot` here) in the vtu file and have it prepended to the cell block name as done here. This page is really outdated, so maybe in the documentation, you can write something based on the following text, if you like it:

==== possible text ====
The naming of the imported `cellBlocks` depends on whether the data array called `regionAttribute` is present in the vtu file or not. This data array is used to define regions in the vtu file and assign the cells to a given region. The `regionAttribute` is a integer and not a string (unfortunately).

- If the vtu file does not contain `regionAttribute`, then all the cells are grouped in a single region, and the cell block names are just constructed from the cell types (`hexahedra`, `wedges`, `tetrahedra`, etc). We have then:
```
cellBlocks="{ hexahedra, wedges, tetrahedra, pyramids }"
```

- If the vtu file contains `regionAttribute`, then the cells are grouped by regions based on their individual (numeric) `regionAttribute`. In that case, the naming convention for the `cellBlocks` is `regionAttribute_elementType`. Assuming for instance that we want the `CellElementRegion` to contain all the cells of type `hexahedra`, `wedges`, and `tetrahedra` from the regions `1` and `3`, we write:
```
cellBlocks="{ 1_hexahedra, 1_wedges, 1_tetrahedra, 1_pyramids, 3_hexahedra, 3_wedges, 3_tetrahedra, 3_pyramids }"
```

Now, coming back to the example illustrated above, let's assume that the top region is identified in the vtu mesh by the `regionAttribute` 1, and that the bottom region is identified with 2. Then, we write
```
<ElementRegions>
  <CellElementRegion
    name="Top"
    cellBlocks="{ 1_hexahedra, 1_wedges, 1_tetrahedra, 1_pyramids }"
    materialList="{ water, rock }"/>
  <CellElementRegion
    name="Bot"
    cellBlocks="{ 2_hexahedra, 2_wedges, 2_tetrahedra, 2_pyramids }"
    materialList="{ water, rock }"/>
</ElementRegions>
```

We remind the user that all the imported `cellBlocks` must be included in one of the `CellElementRegion`s. Specifically, if some cells are meant to be inactive during the simulation, they still have to be included in a `CellElementRegion`. This `CellElementRegion` should simply not be included as a `targetRegion` of any of the solvers involved in the simulation.
==== end of text ====

Also, you can remove the lines just after that:
```
You have to use the following syntax to declare your CellBlocks :

nameOfTheRegionWithinTheMesh_typeOfTheElement
```

Last point, if you want: just above, there is this
```
<Mesh>
  <VTKMesh
    name="MyMeshName"
    file="/path/to/the/mesh/file.vtk"/>
</Mesh>
```
To provide more information, we could have something like the following text:

==== possible text ====
```
<Mesh>
  <VTKMesh
    name="MyMeshName"
    logLevel="1"
    file="/path/to/the/mesh/file.vtk"/>
</Mesh>
```
We advise users to use an absolute path to the mesh file and strongly recommend the use of a `logLevel` of 1 or more to obtain some information about the mesh import. This information contains for example the list of regions that are imported with their names, which is particularly useful to fill the `cellBlocks` field of the `ElementRegions` block (see below). Some information about the imported surfaces is also provided.
==== end of text =====

Reply:
Thanks a lot for these precisions!
I integrated all your suggestions. I just slightly reformulated the cellBlocks part because I prefer to show how the whole ``ElementRegions`` should be defined instead of presenting the ``cellBlocks`` without contextualizing it in an exemple.